data/2024/2024-10-15/meta.yaml

```
@@ -0,0 +1,27 @@
+title: "Southern Resident Killer Whale Encounters"
+article:
+  title: "Web Scraping & Mapping {orcas} Encounters"
+  url: "https://jadeynryan.github.io/orcas/"
+data_source:
+  title: "Center for Whale Research"
+  url: "https://www.whaleresearch.com/"
+images:
+# Please include at least one image, and up to three images
+- file: "l-pod-orcas.png"
+  alt: >
+    Southern Resident Killer Whale L83 carrying the new, emaciated calf L128 
+    across her rostrum. Photo taken by Center for Whale Research under Federal 
+    Permits NMFS PERMIT: "27038/DFO SARA 388."
```

```suggestion
    Southern Resident Killer Whale L83 carrying the new, emaciated calf L128 across her rostrum. Photo taken by Center for Whale Research under Federal Permits NMFS PERMIT: 27038/DFO SARA 388.
```
I don't *think* this matters, but the merge script saw the `PERMIT:` as a YAML header, so I don't want to risk it.